### ISSUE TITLE
[Estrutura] Manual da Marca - Montar as seções do novo manual.

### ISSUE THREAD

**Issue by Vanessavpcs** [SouJunior/products#815]

**Comment by Vanessavpcs** [comment]
@Milenasena segue seu primeiro card de atividade. Na primeira seção de tarefas, você pode ir dando o "check" conforme for passando por elas, aí eu começarei a incluir outras.

Qualquer dúvida pode usar esse espaço para perguntar (só não esqueça de me marcar).

**Comment by Milenasena** [comment]
**Brand book** 

_1 Conceito_

- [ ] naming
- [ ] missão, visão e valores 
- [ ] o que, como e porquê 
- [ ] o que oferecemos | como nosso compromisso | porquê  criar conexões
- [ ] tom de voz da marca


_2 Logo_

- [ ] logo
- [ ] grid
- [ ] respiro
- [ ] positivo e negativo
- [ ] redução
- [ ] não recomendado

_3 Tipografia_

- [ ] tipografia
- [ ] hierarquia de pesos

_4 Cores_

- [ ] cores primárias 
- [ ] cores   @@secundárias

_5 Elementos_

- [ ] grafismos


_6 Aplicações_ 

- [ ] impressos
- [ ] digitais
- [ ] redes sociais
- [ ] landing page


Observação: 
Deixar o link dos arquivos(fonte usada, logo) dentro do manual 

@Vanessavpcs

**Comment by Vanessavpcs** [comment]
@Milenasena estrutura do brand book validada.

**Comment by Vanessavpcs** [comment]
@FelipeMozart sim, estamos fazendo, mas tivemos que parar para dar vazão na campanha do Apoia-se.
